celer/tests/test_mtl.py

```diff
@@ -57,6 +57,7 @@ def test_GroupLasso_MultitaskLasso_equivalence():
     X_data = np.empty([1], dtype=X.dtype)
     X_indices = np.empty([1], dtype=np.int32)
     X_indptr = np.empty([1], dtype=np.int32)
+    # TODO add weights
```

Comment:
FYI: All tests are passed except for this one. 
I need to pass `` weights`` argument to ``dnorm_grp``

Reply:
Great ! When this is done, you'll need to add a test to check that weights behave as expected for GroupLasso. See how it's done for the Lasso here: https://github.com/mathurinm/celer/blob/main/celer/tests/test_lasso.py#L187